packages/pulseaudio.rb

```diff
@@ -36,7 +36,17 @@ class Pulseaudio < Package
   depends_on 'gtk3'
   depends_on 'dbus'
   depends_on 'tdb'
+  depends_on 'cras'
+  depends_on 'orc'
+  depends_on 'jack'
+  depends_on 'avahi'
+  depends_on 'gstreamer'
+  depends_on 'gst_plugins_base'
+  depends_on 'gst_plugins_good'
+  depends_on 'gst_plugins_bad'
+  depends_on 'gst_plugins_bad'
```

Comment:
Am I seeing double lines?

Reply:
I thought about asking you to change to `depends_on 'gst_plugins_ugly'`. :)

Reply:
Good catch on the dupe... I'm having to strongly resist making a Rashomon reference though...